rfcs/0009-vm-syscalls/0009-vm-syscalls.md

```diff
@@ -234,6 +234,38 @@ This syscall would first locate an input in current transaction via `source` and
 
 Specifying an invalid source value here would immediately trigger a VM error, however specifying `output` as the source here would only result in `2` as return value, specifying `current` as source in a *type* script, which doesn't have input, would also result in `2` as return value. Specifying an invalid index value here, would result in `2` as return value, denoting item missing state. Specifying any invalid field will also trigger VM error immediately. Otherwise the syscall would return `0` denoting success state.
 
+### Load Block Via Cell
```

Comment:
I prefer to add a more fine-grained syscall, e.g. `load_header_via_cell`. Reasons:

1. It's unlikely we would allow script to load history transactions in future, because it means script would have access to state in dead cells;
2. Usually scripts only need certain fields in header (e.g. timestamp) instead of a complete header;

The `field` parameter can be more find-grained in `load_header_via_cell`, e.g.

```
- "0": "all"
- "1" : "prevhash"
...
```